MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -545,7 +544,7 @@ void vtkSlicerMarkupsToModelLogic::UpdateOutputHermiteSplineModel(vtkMRMLMarkups
   vtkSmartPointer< vtkMRMLModelNode > modelNode;
     modelNode = markupsToModelModuleNode->GetModelNode();
 
-  int totalNumberOfPoints = markupsToModelModuleNode->GetNumberOfIntermediatePoints()*markupsPointsPolyData->GetNumberOfPoints();
+  int totalNumberOfPoints = markupsToModelModuleNode->GetTubeResolutionLength()*markupsPointsPolyData->GetNumberOfPoints();
```

Comment:
This "resolution length" name is quite confusing. Could you find a better one? TubeSamplingDistance, TubeResolution...